.github/workflows/ci.yml

```
@@ -0,0 +1,205 @@
+          echo "🎯 Ready for deployment"
+        else
+          echo "❌ Quality gate failed"
+          echo "🔧 Test result: ${{ needs.test.result }}"
+          echo "🔨 Build result: ${{ needs.build.result }}"
+          echo "🔒 Security result: ${{ needs.security-scan.result }}"
+          exit 1
+        fi
+
+    - name: Summary
+      if: always()
+      run: |
```

actionlint (1.7.7)

property "performance-check" is not defined in object type {build: {outputs: {}; result: string}; security-scan: {outputs: {}; result: string}; test: {outputs: {}; result: string}}

(expression)